ParselyTrackerTests/EngagedTimeTests.swift

```diff
@@ -1,5 +1,11 @@
 import XCTest
 import os.log
+// FIXME: The Swift package name is different from the name of the framework.
+//
+// This import works when we open the Xcode project, but fails if we open the Package.
+// On fix would be to rename the framework target to ParselyAnalytics, too, then we can use:
+//
+// @testable import ParselyAnalytics
 @testable import ParselyTracker
```

Comment:
I wanted to open an issue in the Parsely repo for this, but the feature is disabled there.